test/smoke_test/bcda_client.go

```diff
@@ -203,22 +225,96 @@ func main() {
 
 					if encrypt {
 						fmt.Println("decrypting the file...")
-						encryptedKey, err := hex.DecodeString(encryptData[path.Base(data[0].Url)])
-						if err != nil {
-							panic(err)
-						}
+						encryptedKey := string(encryptData[path.Base(data[0].Url)])
+
 						privateKeyFile := os.Getenv("ATO_PRIVATE_KEY_FILE")
-						privateKey := getPrivateKey(privateKeyFile)
-						filename = decryptFile(privateKey, encryptedKey, filename)
-						fmt.Printf("writing decrypted file to disk: %s\n", filename)
-					}
 
-					fmt.Println("validating file content...")
-					if !isValidNDJSONFile(filename) {
-						fmt.Println("Error: file is not in valid NDJSON format!")
-						os.Exit(1)
-					}
+						// execute the golang decryptor
+						var cmdOut []byte
+						cmdName := "go"
+						cmdArgs := []string{
+							"run", "../../decryption_utils/Go/decrypt.go",
+							"--file", filename,
+							"--pk", privateKeyFile,
+							"--key", encryptedKey,
+						}
+						fmt.Println("Running the go decryptor externally...")
+						// #nosec
+						if cmdOut, err = exec.Command(cmdName, cmdArgs...).Output(); err != nil {
+							fmt.Fprintln(os.Stderr, "There was an error running the go decryption util: ", err)
+							os.Exit(1)
+						}
+						output := string(cmdOut)
 
+						fmt.Println("validating Go decryptor content...")
+						if !isValidNDJSONText(output) {
+							fmt.Println("Error: file is not in valid NDJSON format!")
+							os.Exit(1)
+						}
+
+						// execute the Python decryptor
+						cmdName = "python"
+						cmdArgs = []string{
+							"../../decryption_utils/Python/decrypt.py",
+							"--file", filename,
+							"--pk", privateKeyFile,
+							"--key", string(encryptedKey),
+						}
+						fmt.Println("Running the Python decryptor externally...")
+
+						// #nosec
+						if cmdOut, err = exec.Command(cmdName, cmdArgs...).Output(); err != nil {
+							fmt.Fprintln(os.Stderr, "There was an error running the Python decryption util: ", err)
+							os.Exit(1)
+						}
+
+						output = string(cmdOut)
```

Comment:
Can we write it to file instead?  I think that will be the standard use-case by clients.

Reply:
except that it isn't.  The enryption examples all pipe the output to a file, none of the utils create a file.  If the utils start making files we can adjust this to use files.

Reply:
Fair enough.  I see this now in the examples:
```
> decrypted_output.ndjson
```

It is, however, a change to how the existing smoke test executes.  I have found it valuable to have the decrypted file be spit out as part of the test execution so that debugging can occur on that decrypted file if a test fails format validation (or if we just want to look at the content).  If we write to string in memory, validate it, and then move on - then we have no output from the test to look at if we need to debug.
Would it make sense to pipe it to a file within the test client so that we have the results of the decryption in case we need to debug it?  
`decrypted_output_go.ndjson`
`decrypted_output_python.ndjson`
`decrypted_output_c.ndjson`

Reply:
In testing I was able to do fmt.Println(output) when I needed to see the files.  I don't really know how we would even start to get at the files in Travis or jenkins if we needed to see them.  Putting the output in the logs is actually the easiest to troubleshoot.

Reply:
But we currently aren't doing `fmt.Println(output)` (and it would really clutter the logs if we did).  So if something fails in the Jenkins pipeline, we have no way of debugging.  
However, if a file is created, we can go onto the EC2 instance and look at it.  (This technique was used heavily when troubleshooting BB DPR integration).  In the legacy decryption mechanism, we are controlling where it is written here: https://github.com/CMSgov/bcda-app/blob/master/test/smoke_test/decrypt_util.go#L60

Reply:
I added in code to print the first bad line it encounters.  This will give us a starting point for debugging.